### PR TITLE
Feature/override cycles

### PR DESCRIPTION
**Pull request by rernst**
- Add override cycles settings to samples field.
- Check sample names on forbidden characters.
- Added production config settings.

### CODE REVIEW

File: portal/forms.py [hunk 1]
```diff
@@ -5,17 +5,23 @@
 import re
 
 from . import lims, app
-import utils
+from . import utils
```
Comment (BasMonkey) [inline portal/forms.py]: Seems like utils could be placed after `app` in the line above (`L7`).

File: portal/utils.py [hunk 2]
```diff
@@ -74,3 +74,8 @@ def transform_sex(value):
             raise ValueError('Ongeldige letter, alleen M, V of O toegestaan.')
     else:
         return value
+
+
+def substrings_in_list(substrings, string):
+    """Check if substrings are in string."""
+    return any(substring in string for substring in substrings)
```
Comment (BasMonkey) [inline portal/utils.py]: Missing new line at the end of file

**Review by BasMonkey — CHANGES_REQUESTED** [review]
See comments (minor changes)

**Comment by rernst** [comment]
Thanks, fixed your comments :)

**Review by BasMonkey — APPROVED** [review]
All OK

**Pull request merged**